.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: false
+language: python
+python:
+  - "2.7"
+  - "3.3"
+install:
```

Comment:
Circle CI has the versions pinned - thats what we want I assume to ensure support with the AHL libraries. I think having Travis use the latest version is good, this way we'll know when something breaks due to a change in another library